sdk/communication/communication-chat/swagger/README.md

```diff
@@ -47,3 +47,13 @@ directive:
   transform: >
     $["x-ms-enum"].modelAsString = false;
 ```
+
+### Set AttachmentType Model as string false
```

Comment:
I think it is turning it into a "teamsInlineImage" because you have turned as string to false here. So I think you remove this configuration and regenerate.

Reply:
no this is the config we need to generate inline enum